packages/web/src/scss/components/RadioField/_RadioField.scss

```diff
@@ -0,0 +1,75 @@
+@use 'theme';
+@use '../../tools/accessibility';
+@use '../../tools/typography';
+
+.RadioField {
+    display: inline-flex;
+    margin-block: theme.$spacing;
+}
+
+.RadioField__input {
+    display: grid;
+    width: theme.$input-width;
+    height: theme.$input-width;
+    margin: theme.$input-margin;
+    font: inherit;
+    color: theme.$input-color-unchecked;
+    background-color: transparent;
+    border: theme.$input-outline-width solid theme.$input-color-unchecked;
+    border-radius: theme.$input-border-radius;
+    appearance: none;
+    transform: translateY(-0.075em);
+    place-content: center;
+
+    &:checked {
+        border-color: theme.$input-color-checked;
+    }
+
+    &:focus-visible {
+        outline: 0;
+        box-shadow: theme.$input-focus-shadow;
+    }
+
+    &::before {
+        content: '';
+        width: theme.$input-inner-dot-width;
+        height: theme.$input-inner-dot-width;
+        border-radius: theme.$input-border-radius;
+        transform: scale(0);
+        transition: theme.$input-transition;
+        box-shadow: theme.$input-box-shadow-before;
```

Comment:
Why not just a background color? 🤔  I somehow don't see the purpose of the `box-shadow`…

Reply:
watch this.. this is inspiration for implementation: https://moderncss.dev/pure-css-custom-styled-radio-buttons/